emcc.py

```diff
@@ -1268,25 +1268,6 @@ def check(input_file):
     if shared.Settings.USE_PTHREADS and shared.Settings.WASM and shared.Settings.ALLOW_MEMORY_GROWTH and shared.Settings.WASM_MEM_MAX == -1:
       exit_with_error('If pthreads and memory growth are enabled, WASM_MEM_MAX must be set')
 
```

Comment:
code here had to be reordered a little, but there is no change in the moved code